### PR TITLE
Ui 785 path

### PR DESCRIPTION
**Pull request by sheela-svmx**
Rename SalesPath to Path 
Refactor Path
Added relavant stories

### CODE REVIEW

**Comment by sheela-svmx** [comment]
@joshsylvester @hanzge @mallik-svmx  @guohonghanusa @kokasvmx

**Comment by sheela-svmx** [comment]
@joshsylvester @mallik-svmx can you please review this one?

File: lib/components/Path/Path.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+  children: null,
+  className: '',
+  items: null,
+  onPathItemClick: null,
+};
+
+class Path extends Component {
+  getPathItems = () => {
+    let pathItems = [];
+    const { children, onPathItemClick, items, ...props } = this.props;
```
[the remark this answers is not in the document]
Comment (sheela-svmx) [reply lib/components/Path/Path.js]: yes we did .. sorry i missed, my bad. will update

File: lib/components/Path/Path.test.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { shallow, mount } from 'enzyme';
+import { Path, PathItem } from './../Path';
```
Comment (hanzge) [inline lib/components/Path/Path.test.js]: use `'../Path'` to follow our pattern

File: lib/components/Path/PathItem.js [hunk 3]
```diff
@@ -0,0 +1,89 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import { Icon } from './../Icon';
```
Comment (hanzge) [inline lib/components/Path/PathItem.js]: `'../Icon'`

File: lib/components/Path/PathItem.js [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+      `slds-is-${type}`,
+      className,
+    );
+    const tabIndex = type === 'current' ? 0 : -1;
+    const completedText = completedTitle || 'Stage Complete';
+
+    return (
+      <li className={pathItemClassName} role="presentation">
+        <a
+          className="slds-tabs--path__link"
```
Comment (hanzge) [inline lib/components/Path/PathItem.js]: Add our className `Path__link`

File: lib/components/Path/PathItem.js [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+          aria-selected="false"
+          tabIndex={tabIndex}
+          role="tab"
+          onClick={this.handleItemClick}
+        >
+          <span className="slds-tabs--path__stage">
```
Comment (hanzge) [inline lib/components/Path/PathItem.js]: Add `Path__stage`

File: lib/components/Path/PathItem.js [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+            <Icon category="utility" icon="check" size="x-small" />
```
[the remark this answers is not in the document]
Comment (sheela-svmx) [reply lib/components/Path/PathItem.js]: @joshsylvester so there are checkmark for paths with type complete,  They are always rendered and is controlled by the styles

**Comment by sheela-svmx** [comment]
@joshsylvester updated

**Comment by joshsylvester** [comment]
two approvals, merging.

**Pull request merged**